frontend/src/components/common/Filter/components/FilterContent/components/FilterMenu/components/FilterMenuItems/index.tsx

```diff
@@ -1,42 +1,47 @@
-import { MenuItem } from "@blueprintjs/core";
-import { IconNames } from "@blueprintjs/icons";
+import { Icon } from "czifui";
+import React from "react";
+import { ListItemIcon, ListItemText } from "@mui/material";
 import {
   CATEGORY_FILTER_ID,
   OnFilterFn,
   SelectCategoryValueView,
 } from "src/components/common/Filter/common/entities";
-import { SelectionIcon } from "../../../../../../common/style";
-import { MenuItemWrapper } from "./style";
+import { ListItem } from "./style";
 
 interface Props {
   categoryFilterId: CATEGORY_FILTER_ID;
-  isMultiselect: boolean;
   menuItems: SelectCategoryValueView[];
   onFilter: OnFilterFn;
 }
 
 export default function FilterMenuItems({
   categoryFilterId,
-  isMultiselect,
   menuItems,
   onFilter,
 }: Props): JSX.Element {
+  // (clevercanary): For optimal rendering performance for the PUBLICATION_AUTHORS category filter, the SDS ListItem
```

Comment:
Rendering performance of the `PUBLICATION_AUTHORS` category filter is considerably slower using SDS/Mui `ListItem` than Blueprint's equivalent.

Using a virtualised list such as `FixedSizeList` from `react-window` is not a simple exercise - we would need to measure after fonts have loaded the longest list item to determine minimum menu width. Alternatively, setting menu width would require the use of `VariableSizeList` from `react-window` with each item's size having to be calculated to ensure the item is not wrapped...

For simplicity we are rendering the `ListItem` as a basic `li` element and in doing so, improve rendering performance considerably.